natlas-agent/chrome.json

```diff
@@ -0,0 +1,1535 @@
+{
```

Comment:
I think this can be excluded in the .dockerignore file because we don't need this in the container itself.